packages/edit-site/src/components/block-editor/index.js

```diff
@@ -39,13 +41,32 @@ export default function BlockEditor( { settings: _settings } ) {
 			},
 		};
 	}, [ canUserCreateMedia, _settings ] );
-	const [ blocks, setBlocks ] = useState( [] );
+	const [ content, _setContent ] = useEntityProp(
```

Comment:
Why underscore?

Reply:
To avoid shadowing.

I'm replacing this with `useEntityBlockEditor` in the follow-up PR. Can we ignore it for now?